core/drivers/firewall/stm32_etzpc.c

```diff
@@ -153,6 +156,44 @@ static enum etzpc_decprot_attributes etzpc_binding2decprot(uint32_t mode)
 	}
 }
 
+static void
+sanitize_decprot_config(uint32_t decprot_id __maybe_unused,
+			enum etzpc_decprot_attributes attr __maybe_unused)
```

Comment:
attr is always used in the switch, please remove __maybe_unused

Reply:
When `CFG_STM32MP15` is disabled, the function is empty.

Reply:
Absolutely, my bad